wip-trampoline.md

```diff
@@ -229,33 +229,48 @@ TODO: use a TLV stream for `node_update`.
 TODO: should use an odd type (261?) because it's safe to ignore if unsupported?
 
 It has a structure similar to the `channel_update` message but is channel
-agnostic. Trampoline nodes need to estimate a `cltv_expiry_delta` and `fees`
-that would allow them to route to any other trampoline node. `Node_update`s
-should be relayed the same way `channel_update`s are relayed.
+agnostic. `Node_update`s should be relayed the same way `channel_update`s are
+relayed (staggered broadcast).
+
+If a node is not willing to relay trampoline payments, it will simply never
+send a `node_update`. This lets the network discriminate nodes that are able to
+relay trampoline payments from nodes that use it but simply as sender or
+destination.
 
 ### Fees and CLTV estimation
 
-TODO: detail heuristics that can be used by trampoline nodes to estimate cltv
-and fees.
+Trampoline nodes need to estimate a `cltv_expiry_delta` and `fee` that would
```

Comment:
Trampoline nodes should be encouraged to not be constrained and keep a full view of the topology.

Reply:
Yes definitely. I think this incentivizes them to do it because it's the best way for them to be more economically viable.